egs/wsj/s5/steps/nnet3/get_egs.sh

```diff
@@ -80,6 +89,15 @@ if [ $# != 3 ]; then
   echo "  --num-frames-diagnostic <#frames;4000>           # Number of frames used in computing (train,valid) diagnostics"
   echo "  --num-valid-frames-combine <#frames;10000>       # Number of frames used in getting combination weights at the"
   echo "                                                   # very end."
+  echo "  --left-shift-window <bool;false>                 # If false, the last example extracted from an utterance would"
+  echo "                                                   # be padded with the final frame to get frames_per_eg frames."
+  echo "                                                   # If true, the starting frame-index of the last example would be"
+  echo "                                                   # shifted to the left to get frames_per_eg frames."
+  echo " --ivector-interval <#frames;0>                    # Specify the interval between two consecutive frames that we want to"
```

Comment:
<pre><code>
echo " --ivector-interval <#frames;0>      # Default 0 means having only one ivector for each example." 
echo "                  # If non-zero, this value is the interval between two consecutive ivectors to be dumped in the example"
echo "                  # It is recommended to have an ivector-interval of at most 10, for good results."
</code></pre>